tests/unit/amo/components/TestSearchPage.js

```diff
@@ -48,44 +48,49 @@ describe('<SearchPage />', () => {
 
   it('renders a Paginate', () => {
     const root = render();
-    const paginator = findByTag(root, Paginate);
-    expect(paginator.props.count).toEqual(80);
-    expect(paginator.props.currentPage).toEqual(3);
-    expect(paginator.props.pathname).toEqual('/search/');
-    expect(paginator.props.queryParams).toEqual({ q: 'foo' });
+    const paginator = root.find(Paginate);
+    expect(paginator.prop('count')).toEqual(80);
```

Comment:
> Will expect(80).toEqual('80'); pass?

No you'd get: 

<img width="645" alt="repl_it_-_jest_compiler" src="https://user-images.githubusercontent.com/1514/26968363-acaf3774-4cf9-11e7-9c66-a0ce9c678e49.png">

`toBe` checks identity through using `===` so if you were comparing two objects it would only pass if the objects were identical.

The second thing is the complexity in the code behind `isEqual()` see https://github.com/facebook/jest/blob/master/packages/jest-matchers/src/jasmine-utils.js#L58 which is the function behind it. Whether this amounts to anything that would actually result in a meaningful slowdown remains to be seen.

From the perspective of just running up the tests using `toEqual()` most of the time wouldn't cause problems. Maybe we don't worry about it?